src/js/post-911-gib-status/reducers/index.js

```diff
@@ -0,0 +1,25 @@
+import set from 'lodash/fp/set';
+
+const initialState = {
+  enrollmentData: null,
+  available: false
+};
+
+function post911GIBillStatus(state = initialState, action) {
```

Comment:
We may want to pick either GIBillStatus or GibStatus throughout to keep us sane, but not critical